ecosystem.md

```diff
@@ -11,19 +11,141 @@ many other data processing projects and frameworks.
 
 <p>Connectors provide code for interfacing with various third-party systems.</p>
 
-<p>Currently these systems are supported:</p>
-
-<ul>
-  <li><a href="{{site.docs-stable}}/dev/connectors/kafka.html" target="_blank">Apache Kafka</a> (sink/source)</li>
-  <li><a href="{{site.docs-stable}}/dev/connectors/elasticsearch.html" target="_blank">Elasticsearch 1.x / 2.x / 5.x / 6.x</a> (sink)</li>
-  <li><a href="{{site.docs-stable}}/dev/connectors/filesystem_sink.html" target="_blank">HDFS</a> (sink)</li>
-  <li><a href="{{site.docs-stable}}/dev/connectors/rabbitmq.html" target="_blank">RabbitMQ</a> (sink/source)</li>
-  <li><a href="{{site.docs-stable}}/dev/connectors/kinesis.html" target="_blank">Amazon Kinesis Streams</a> (sink/source)</li>
-  <li><a href="{{site.docs-stable}}/dev/connectors/twitter.html" target="_blank">Twitter</a> (source)</li>
-  <li><a href="{{site.docs-stable}}/dev/connectors/nifi.html" target="_blank">Apache NiFi</a> (sink/source)</li>
-  <li><a href="{{site.docs-stable}}/dev/connectors/cassandra.html" target="_blank">Apache Cassandra</a> (sink)</li>
-  <li><a href="https://github.com/apache/bahir-flink" target="_blank">Redis, Flume, and ActiveMQ (via Apache Bahir)</a> (sink)</li>
-</ul>
+<p>
+  The following table lists the connectors that are available in the Flink ecosystem.
+  Connector authors are welcome to add their connector to this list via a Github
+  Pull Request. See <a href="{{site.docs-stable}}/improve-website.html">Improve the Website</a>
+  for more details.
+</p>
+
+<table class="table table-bordered">
+  <tr>
+    <th>System Name</th>
+    <th>Connector Type</th>
+    <th>Location</th>
+    <th>License</th>
+    <th>Last released</th>
+    <th>Compatible Flink Versions</th>
+    <th>Maintained By</th>
+  </tr>
+  <tr>
+    <td><a href="{{site.docs-stable}}/dev/connectors/kafka.html" target="_blank">Apache Kafka</a></td>
+    <td>sink/source</td>
+    <td>Apache Flink</td>
+    <td>Apache 2.0</td>
+    <td></td>
+    <td>1.7.x</td>
+    <td>Apache Flink</td>
+  </tr>
+  <tr>
+    <td><a href="{{site.docs-stable}}/dev/connectors/elasticsearch.html" target="_blank">Elasticsearch 1.x / 2.x / 5.x / 6.x</a></td>
+    <td>sink</td>
+    <td>Apache Flink</td>
+    <td>Apache 2.0</td>
+    <td></td>
+    <td>1.7.x</td>
+    <td>Apache Flink</td>
+  </tr>
+  <tr>
+    <td><a href="{{site.docs-stable}}/dev/connectors/filesystem_sink.html" target="_blank">HDFS</a></td>
```

Comment:
We could replace `HDFS` by `File Systems (HDFS, S3, and others)` and make others a link.

Regarding the encodings, I think we should list them in a separate section. Putting them here would clutter the connector list, but I think it is important to show that Flink supports different encodings like Parquet and ORC. Having a file system connector does not help a lot if the encoding that a user needs is not supported.

Reply:
It is absolutely necessary to mention the supported encoding information. But this seems something specific to particular file systems, instead of applicable to all connectors. So would it be better to have the encoding information in the individual pages of those specific systems?

Reply:
The supported encodings are independent of the file system. However, they depend on the connector implementation (Flink API). For example, we support ORC as source format for DataSet and Table API (and probably also DataStream via continuous file source) but not as sink format.